tools/bug-report/pkg/content/content.go

```diff
@@ -150,6 +150,22 @@ func GetIstiodInfo(p *Params) (map[string]string, error) {
 	return ret, nil
 }
 
+// GetProxyInfo returns internal proxy debug info.
+func GetProxyInfo(p *Params) (map[string]string, error) {
+	if p.Namespace == "" || p.Pod == "" {
+		return nil, fmt.Errorf("getIstiodInfo requires namespace and pod")
+	}
+	ret := make(map[string]string)
+	for _, url := range common.ProxyDebugURLs(p.ClusterVersion) {
+		out, err := kubectlcmd.Exec(p.Client, p.Namespace, p.Pod, common.ProxyContainerName, fmt.Sprintf(`curl http://localhost:15000/%s`, url), p.DryRun)
```

Comment:
Should this use `pilot-discovery request GET` instead, due to distroless?

Reply:
Oh, John beat me to it.

Reply:
that won't work for envoy, here we are doing a curl from proxy container. does the distroless comment apply to envoy too?

Reply:
it won't be pilot-discovery but it still applies. Even still, the best move is to do what istioctl does since that isthe well tested path and we chose it for a reason

Reply:
using envoydo now, per comment above.